tests/e2e/config/setup.js

```diff
@@ -0,0 +1,129 @@
+// From https://github.com/WordPress/gutenberg/blob/master/packages/e2e-tests/config/setup-test-framework.js
+
+/**
+ * External dependencies
+ */
+// import { get } from 'lodash';
+// import jest, { beforeAll } from 'jest';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	/*activatePlugin, clearLocalStorage, */
+	setBrowserViewport,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Environment variables
+ */
+const { PUPPETEER_TIMEOUT } = process.env;
+
+/**
+ * Internal dependencies
+ */
+// import { init } from './jasmine-fail-fast';
+
+// Add custom Jasmine reporter to skip tests in failing describe
+// const jasmineEnv = jasmine.getEnv();
+// jasmineEnv.addReporter( init() );
+
+/**
+ * Set of console logging types observed to protect against unexpected yet
+ * handled (i.e. not catastrophic) errors or warnings. Each key corresponds
+ * to the Puppeteer ConsoleMessage type, its value the corresponding function
+ * on the console global object.
+ *
+ * @type {Object<string,string>}
+ */
+// const OBSERVED_CONSOLE_MESSAGE_TYPES = {
```

Comment:
can we remove all of the commented code in this file? Or is there some reason to keep it?

Reply:
Removed in 531a924